Cargo.toml

```diff
@@ -27,12 +27,19 @@ harness = false
 [features]
 default = ["blake3/default", "std", "winter_crypto/default", "winter_math/default", "winter_utils/default"]
 std = ["blake3/std", "winter_crypto/std", "winter_math/std", "winter_utils/std"]
+serde = ["winter_math/serde", "dep:serde", "dep:serde_arrays", "serde/alloc"]
 
 [dependencies]
 blake3 = { version = "1.4", default-features = false }
 winter_crypto = { version = "0.6", package = "winter-crypto", default-features = false }
 winter_math = { version = "0.6", package = "winter-math", default-features = false }
 winter_utils = { version = "0.6", package = "winter-utils", default-features = false }
+serde = { version = "1.0", features = [ "derive" ], optional = true, default-features = false }
+serde_arrays = { version = "0.1", optional = true }
```

Comment:
serde doesn't support `[T; C]` where `C` is a compile time constant. ref [#1937](https://github.com/serde-rs/serde/issues/1937).

This crate implements code to work around that deficiency.